### PR TITLE
ci: Upload node binary to bintray on builds

### PR DESCRIPTION
**Pull request by geigerzaehler**
We upload the node binaries to bintray on every build. The download URL is documented in the README

### CODE REVIEW

File: ci/run [hunk 1]
```diff
@@ -87,3 +87,40 @@ else
   echo "$cache_key" > "$target_cache_key_file"
   echo "Size of $target_cache is $(du -sh "$target_cache" | cut -f 1)"
 fi
+
+# Upload the node binary to bintray using the commit hash as the version.
+function upload_to_bintray () {
+  declare -r api_url="https://bintray.com/api/v1"
```
Comment (kim) [inline ci/run]: Bash zealot drive-by comment: `local` accepts the same flags as `declare`, but restricts the variable visibility to the function.

Comment (geigerzaehler) [reply ci/run]: TIL
